packages/arb-token-bridge-ui/src/token-bridge-sdk/oftV2Abi.ts

```diff
@@ -0,0 +1,381 @@
+export const oftV2Abi = [
```

Comment:
Exporting it in TS file rather than JSON means we can use `as const` and have typing from wagmi in `writeContract`